lib/drivers/docs/medtronic.md

```diff
@@ -0,0 +1,229 @@
+  - `[x]` final (most recent) basal
+    - `*[-]` basal rate interval with a start time, duration "guessed" from settings, rate delivered, and an annotation re: the "guessed" duration OR
+    - `[ ]` basal rate interval with a start time and rate, no (= zero) duration
+
+Device-specific? (Add any device-specific notes/additions here.)
+
+#### Boluses
+
+  - `[x]` normal bolus
+    - `[x]` amount of insulin delivered
+    - `[x]` amount of insulin delivery programmed (if differs from actual delivery, in case of bolus interruption, cancellation, etc.)
+  - `[x]` extended bolus
+    - `[x]` amount of insulin delivered
+    - `[x]` duration of insulin delivery
+    - `[x]` amount of insulin delivery programmed (if differs from actual delivery, in case of bolus interruption, cancellation, etc.)
+    - `[x]` duration of insulin delivery programmed (if differs from actual duration, in case of bolus interruption, cancellation, etc.)
+    - `[ ]` extended bolus that crosses midnight is split into two records
+  - `[x]` combo/dual bolus
+    - `[x]` amount of insulin delivered - immediate (normal)
+    - `[x]` amount of insulin delivered - extended
+    - `[x]` duration of extended insulin delivery
+    - `[x]` amount of immediate insulin delivery programmed (if differs from actual delivery, in case of bolus interruption, cancellation, etc.)
+    - `[x]` amount of extended insulin delivery programmed (if differs from actual delivery, in case of bolus interruption, cancellation, etc.)
+    - `[x]` duration of extended insulin delivery programmed (if differs from actual duration, in case of bolus interruption, cancellation, etc.)
+    - `[ ]` extended portion of combo bolus that crosses midnight is split into two records
+  - bolus cancellations/interruptions
+    - `[x]` represented by a separate event in the device's data log OR
+    - `[ ]` result in modifications to a bolus event in the device's data log
+  - `[x]` link to "wizard"/calculator entry (via log entry ID or similar)
+
+No Tidepool data model yet:
+
+  - bolus cancellations/interruptions
+    - `[ ]` agent/reason for bolus cancellation
+
+Device-specific? (Add any device-specific notes/additions here.)
+
+#### CBG
+
+(See [the CGM checklist](CGMChecklist.md) instead.)
+
+#### Device Events
+
+  - alarms:
+    - `[x]` low insulin
+    - `[ ]` no insulin
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[x]` low power
+    - `[x]` no power
+        - `[x]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[ ]` occlusion
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[x]` no delivery
+        - `[x]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[x]` auto-off
+        - `[x]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[?]` over limit (i.e., max bolus exceeded through override)
```

Comment:
@eluhrs Just check if it's at all possible to create an "over limit" alarm manually on the pump. I tried but failed: The Medtronic UI is pretty good at preventing you from exceeding your limits. If you do manage, please write down the steps so that I can try to reproduce too.